database/step.go

```diff
@@ -126,6 +126,32 @@ func (c *client) GetStepImageCount() (map[string]float64, error) {
 	return counts, err
 }
 
+// GetStepStatusCount gets a count of an images occurance in the database.
+func (c *client) GetStepStatusCount() (map[string]float64, error) {
+	logrus.Tracef("Counting the total of each status for steps in the database")
+
+	type statusCount struct {
+		Status string
+		Count  int
+	}
+
+	// variable to store query results
+	statuses := new([]statusCount)
```

Comment:
This could optionally be changed to `s` since its value is struct not primitive.
https://github.com/golang/go/wiki/CodeReviewComments#variable-names